oneandone/src/test/java/org/apache/jclouds/oneandone/rest/features/UserApiLiveTest.java

```diff
@@ -0,0 +1,162 @@
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jclouds.oneandone.rest.features;
+
+import java.util.List;
+import org.apache.jclouds.oneandone.rest.domain.User;
+import org.apache.jclouds.oneandone.rest.domain.options.GenericQueryOptions;
+import org.apache.jclouds.oneandone.rest.internal.BaseOneAndOneLiveTest;
+import org.testng.Assert;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+@Test(groups = "live", testName = "UserApiLiveTest")
+public class UserApiLiveTest extends BaseOneAndOneLiveTest {
+
+   private User currentUser;
+   private List<User> users;
+
+   private UserApi userApi() {
+
+      return api.userApi();
+   }
+
+   @BeforeClass
+   public void setupTest() {
+      currentUser = userApi().create(User.CreateUser.builder()
+              .name("jcloudsuser5")
+              .description("description")
+              .email("test@gtmail.com")
+              .password("test123!")
+              .build());
+   }
+
+   @Test
+   public void testList() {
+      users = userApi().list();
+
+      Assert.assertTrue(users.size() > 0);
+   }
+
+   @Test
+   public void testListWithOption() {
+      GenericQueryOptions options = new GenericQueryOptions();
+      options.options(0, 0, null, currentUser.name(), null);
+      List<User> resultWithQuery = userApi().list(options);
+      if (resultWithQuery.isEmpty()) {
+         return;
+      }
+      currentUser = resultWithQuery.get(0);
+
+      Assert.assertTrue(resultWithQuery.size() > 0);
+   }
+
+   @Test
+   public void testGet() {
+      User result = userApi().get(currentUser.id());
+
+      assertEquals(result.id(), currentUser.id());
+   }
+
+   @Test(dependsOnMethods = "testGet")
+   public void testUpdate() throws InterruptedException {
+      //this test will fail too since its not allowed to update other users from the API
+//      String updatedDescription = "updatejclouds";
+//
+//      User updateResult = userApi().update(currentUser.id(), User.UpdateUser.builder()
+//              .description(updatedDescription)
+//              .build());
+//
+//      assertEquals(updateResult.description(), updatedDescription);
```

Comment:
I agree this should not be our highest priority.
Moving forward, do you think the existing APIs cover the funcionality to implement the ComputeService abstraction?